ocfweb/lab_reservations/reserve.py

```diff
@@ -0,0 +1,186 @@
+from collections import namedtuple
+from datetime import datetime
+
+import ocflib.misc.validators
+from django import forms
+from django.shortcuts import render
+from jinja2 import Environment
+from jinja2 import PackageLoader
+from ocflib.account.search import user_attrs_ucb
+from ocflib.misc.mail import send_mail
+
+from ocfweb.auth import calnet_required
+from ocfweb.component.forms import wrap_validator
+
+
+JINJA_MAIL_ENV = Environment(loader=PackageLoader('ocfweb', ''))
+
+
+class NewReservationRequest(namedtuple(
+    'NewReservationRequest', [
+        'real_name',
+        'contact_email',
+        'student_group',
+        'reason',
+        'date',
+        'starttime',
+        'endtime',
+    ],
+)):
+    """Request for reservation request.
+    :param real_name:
+    :param contact_email:
+    :param student_group:
+    :param reason:
+    :param date:
+    :param starttime:
+    :param endtime:
```

Comment:
I don't really think giving all the parameters to the class here gives any information, I mean they're just a couple lines this if anyone wants to actually look at them. Instead, maybe include here more details on what this class represents, when one is created, information like that, and remove this information at least since it's not useful and just boilerplate.

(this is essentially what @chriskuehl said below about the docstring, but I typed all of this out, so I'm going to leave it. I'd agree with him that using the `namedtuple` constructor without inheriting from it and using `_asdict()` would be cleaner here)